src/styles/molecules/_tabs.scss

```diff
@@ -0,0 +1,34 @@
+.tab{
+  width: 40%;
```

Comment:
this shouldn't have any set width and should be a block. The tab component will always fill it's parent :)